can_tools/scrapers/base.py

```diff
@@ -162,6 +166,7 @@ class DatasetBase(ABC):
     location_type: Optional[str]
     base_path: Path
     source: str
+    demographic_data: bool = False
```

Comment:
Adding this `demographic_data` flag to all demographic scrapers only for use in `can_tools/__init__.py` feels a little not-great, but I couldn't think of an easier/more elegant way. 

I thought about selecting all the demographic scrapers by looking for `Demographics`, `Age`, `Race` etc. substrings in the class names, but that felt much less stable.